packages/axios-extension/src/auth/connection.ts

```diff
@@ -35,8 +35,8 @@ export class Cookies {
      * @returns cookies object
      */
     public addCookie(cookieString: string): Cookies {
-        const cookie = cookieString.split(';');
-        const [key, ...values] = cookie[0]?.split('=');
+        const cookie = cookieString?.split(';') || [];
```

Comment:
Based on the function signature `cookieString` is always set, why do we need to do the null check then in line 38?

Reply:
And Sonar agrees with me ;)
It only complain about line 39 https://sonarcloud.io/code?id=SAP_open-ux-tools&selected=SAP_open-ux-tools%3Apackages%2Faxios-extension%2Fsrc%2Fauth%2Fconnection.ts&line=39

Reply:
@tobiasqueck: I agree that it is not required if used like in the tests. However, since `Cookies` is a public type and a consumer could directly use `addCookie`, this additional safety may be required.

Reply:
Actually, I take that back. It looks like `split` works fine with an empty string. The result of `''.split[';']` results in `['']` and the check in line 39 should then be sufficient. I'll update the code.

Reply:
Reverted changes in line 38.